ckanext/canada/scripts.py

```diff
@@ -266,13 +267,18 @@ def resubmit_empty_datastore_resources(resource_id=None, verbose=False, quiet=Fa
                 continue
             if resource_id in datastore_tables:
                 try:
-                    count = _get_datastore_count(context, resource_id, verbose=verbose, status=status, max=max)
-                    if int(count) == 0:
-                        if verbose:
-                            click.echo("%s/%s -- Resource %s has %s rows in DataStore. Let's fix this one..." % (status, max, resource_id, count))
+                    if empty_only:
+                        count = _get_datastore_count(context, resource_id, verbose=verbose, status=status, max=max)
+                        if int(count) == 0:
```

Comment:
This pattern suggests that _get_datastore_count should already be returning an int.